transformers/index.html

```diff
@@ -10,100 +10,98 @@
+        </svg>
+      </a>
+      transformers
+      <span>
+      </span>
+      <span id='latest-version' class="version">
+        3.0
+      </span>
+      <span id='latest-main-version' hidden>
+        3.0
+      </span>
     </section>
-    <pre id='installdanger' hidden><button class="danger-button" disabled>DANGER ! A higher version of <i>transformers</i> already exists on PyPi</button></pre>
-    <pre id='installcmd'><code>pip install transformers --extra-index-url https://astariul.github.io/github-hosted-pypi/</code></pre>
-    <a href="git+https://github.com/huggingface/transformers@v3.0.2#egg=transformers-3.0">
-    </a>
-    <hr/>
+    <pre id='installdanger'
+      hidden><button class="danger-button" disabled>DANGER ! A higher version of <i>transformers</i> already exists on PyPi</button></pre>
+    <pre
+      id='installcmd'><code>pip install transformers --extra-index-url https://astariul.github.io/github-hosted-pypi/</code></pre>
+    <hr />
     <div class="row">
-     <div class="three columns">
-      <button onclick="location.href='https://github.com/huggingface/transformers'">
-       Homepage
-      </button>
-      <p class="elem">
-       <b>
-        Author :
-       </b>
-       Nicolas Remond
-      </p>
-     </div>
-    <div class="nine columns">
-     <h6 class="text-header">
-      Description
-     </h6>
-     <p>
-      This is a (safe) example of a package vulnerable to supply chain attacks.
-      <br>
-      Here we registered a private package called <i>transformers</i>. But another package with the exact same name and a higher version is registered in the public PyPi index. Running the install command would install the package registered there (which might be malicious), not my private package as intended.
-      <br>
-      A warning is displayed instead of the install command.
-     </p>
+      <div class="three columns">
+        <b>
+          Project links :
+        </b>
+        <a href="https://github.com/huggingface/transformers" target="_blank" id="repoHomepage">
+          <button>
+              Github Repository
+          </button>
+      </a>
+        <p class="elem">
+          <b>
+            Author :
+          </b>
+          Nicolas Remond
+        </p>
+        <section class="versions" id="versions">
+          <div onclick="load_readme('3.0', scroll_to_div=true)" id="3.0">
+            <a>
+              3.0
+            </a>
+            <a href="git+https://github.com/huggingface/transformers@3.0#egg=transformers-3.0">
+            </a>
+          </div>
+        </section>
+      </div>
+      <div class="nine columns" id="description_pkg">
+        <h6 class="text-header">
+          Description
+        </h6>
+        <p id="markdown-container" class="readme-block">
+        </p>
+      </div>
     </div>
-   </div>
   </div>
+  <footer class="footer">
+    (c) All Rights Reserved to Whatever Ltd. 2023.
+   </footer>
```

Comment:
Should we include this footer in the template repository ?

Reply:
The footer is generated in all packages because i create this script [update_pkgs](https://github.com/pabnas/github-hosted-pypi/blob/general_fix_and_update_styles/update_pkgs.py) that deletes and register all packages so applies the changes on the template without needing to do it manually. so removing this footer only in this package has to be done after running the scripts.

Reply:
I mean do we need this footer in all packages ?

I can understand the need for this footer in people's website, but not everyone need it. So maybe we don't need to add it here in the template repository, what do you think ?

Reply:
ok, i think is better to leave it only in the index page

Reply:
I mean maybe we should remove it completely ? Even from the index page ?

Reply:
yeah is for the best removing that, if someone needs the footer it will have to create it anyways.